.circleci/config.yml

```diff
@@ -315,8 +335,74 @@ jobs:
           name: "Run Tests"
           command: yarn test
 
+  bootstrap-with-experimental-react:
+    executor: node
+    parameters:
+      version:
+        type: string
+        default: "next"
+    steps:
+      - checkout
+      - run:
+          name: "Update React to prerelease"
+          command: "REACT_CHANNEL=<< parameters.version >> node ./scripts/upgrade-react"
+      - run: yarn install
+      - run: yarn bootstrap -- concurrency=2
+      - run: yarn list react
+      - persist_to_workspace:
+          root: ./
+          paths:
```

Comment:
Maybe this is the default?

Reply:
https://circleci.com/docs/2.0/configuration-reference/#persist_to_workspace

both keys are required, no defaults listed :\